shared/constants/platform.desktop.js

```diff
@@ -123,24 +123,31 @@ function findCacheRoot(): string {
 }
 
 function logDir(): string {
+  // See LogDir() functions in go/libkb/home.go.
+  //
+  // TODO: darwin and win32 cases are inconsistent with their LogDir()
+  // counterparts. Fix this.
   switch (process.platform) {
     case 'darwin':
       return `${getenv('HOME', '')}/Library/Logs`
     case 'linux':
-      const linuxDefaultRoot = `${getenv('HOME', '')}/.local/share`
-      return `${getenv('XDG_DATA_HOME', linuxDefaultRoot)}/${envedPathLinux[runMode]}`
+      return findCacheRoot()
```

Comment:
`~/.local/share` is the wrong place to put logs. Note that this also changes `jsonDebugFileName`